{"name": "test_ConsultationOrders[2020-02-1]", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../api/api.yaml'", "trace": "cls = <class 'gm.testcase.test_api.Test_api'>\n\n    @classmethod\n    def setup_class(cls):\n>       cls.login = Testapi().login()\n\ntestcase/test_api.py:13: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\napi/api_face.py:16: in __init__\n    self.data = self.api_load(\"../api/api.yaml\")\napi/base_api.py:33: in api_load\n    return self.yaml_load(path)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\ncls = <class 'gm.api.api_face.Testapi'>, path = '../api/api.yaml'\n\n    @classmethod\n    def yaml_load(cls, path) -> list:\n>       with open(path) as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: '../api/api.yaml'\n\napi/base_api.py:28: FileNotFoundError"}, "parameters": [{"name": "year", "value": "'2020'"}, {"name": "month", "value": "'02'"}, {"name": "page", "value": "'1'"}], "start": 1588835843123, "stop": 1588835843123, "uuid": "84c07c25-014e-4474-a515-d2d168756fd6", "historyId": "74a486ea1d2435525746ae63cf9910e8", "testCaseId": "90072c1d28b48bbf14e9fbceea841c64", "fullName": "testcase.test_api.Test_api#test_ConsultationOrders", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "Test_api"}, {"name": "host", "value": "dengyingdeMacBook-Pro.local"}, {"name": "thread", "value": "3477-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_api"}]}